src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,10 +11,14 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
+    
+    public String GetFeedback() {
```

Comment:
do take note that method names start with lower case alphabet followed by camel casing convention. Would be good if you can add a method header comment to this new method